code/src/App.js

```diff
@@ -1,9 +1,47 @@
-import React from 'react'
+import React from "react";
+import { Provider } from "react-redux";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { compose, createStore, combineReducers } from "@reduxjs/toolkit";
+import persistState from "redux-localstorage";
+import { projects } from "reducers/projects";
+import { colors } from "reducers/colors";
+import { icons } from "reducers/icons";
+import Home from "components/Home";
+import Error from "components/Error";
+import Project from "components/Project";
+import Header from "components/Header";
+import NewProject from "components/NewProject";
+import ErrorBoundary from "components/ErrorBoundary";
+
+const enhancer = compose(persistState());
```

Comment:
Nice seeing you challenge yourself with adding new tools to your project. Maybe add a small comment for the reviewer  that explains the use of these?

Reply:
The enhancer is for localstorage which was in the "Common Pitfalls" section of the project's brief.

Error boundaries I don't completely understand myself - I was getting some errors in my console log, and had to read some guides. They basically check for errors being passed into the component as far as I remember, and then can render an error component instead. Please don't quote me on it - I was very confused while working on it, and erased it from my memory as soon as I got it to work 😅